### ISSUE TITLE
Implement embedding typeform/ custom url

### ISSUE THREAD

**Issue by katjam** [neontribe/contemplating-action#73]
## Description
Look at options for custom url or embed of Typeform survey

## Acceptance Criteria
- [ ] We understand the methods/ tradeoffs for integrating typeform so that we can make an informed choice.

**Comment by katquatermass** [comment]
Currently blocked by sorting out typeform URL (that does have an issue number, but I can't find it right now, will update when I can) #51

**Comment by katjam** [comment]
https://contemplating-action.typeform.com/to/zUOTYA

**Comment by katjam** [comment]
Won't fix until/ unless we need embedded for for one of the charity sites
